### ISSUE TITLE
Error loading image in talk app

### ISSUE THREAD

**Issue by filipdot** [nextcloud/talk-desktop#992]
### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are affected by the same issue.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments.

---

## Steps to reproduce
1. Install talk desktop on windows 10
2. Log in
3. Recive photo from somebody in message
4. get error

### Expected behaviour
I should see photo, in broweser works ok.
in broweser i see file (but idk why previe doesn't work)
![Image](https://github.com/user-attachments/assets/290d58c9-0222-4e7b-ad02-5fb75d78cf99)
![Image](https://github.com/user-attachments/assets/dde24f47-edab-45e6-8d6d-e54865f3be22)

### Actual behaviour
"Could not load the file.
![Image](https://github.com/user-attachments/assets/8e214521-76ea-4448-8b81-fa79d28fa4f7)

## Diagnosis and logs

<details>
<summary>Diagnosis report</summary>
### Diagnosis report

| **Nextcloud Talk Desktop**      |                                                                         |
| ------------------------------: | ----------------------------------------------------------------------- |
| **Version**                     | v1.0.0                                                                  |
| **Built-in Talk version**       | v20.1.0                                                                 |
| **Operating system**            | Windows_NT 10.0.19045 (Windows 10 Pro)                                  |
| **Executable Path**             | C:\Users\Filip\AppData\Local\NextcloudTalk\app-1.0.0\Nextcloud Talk.exe |
| ****Connected to****            | -                                                                       |
| **Nextcloud server version**    | 30.0.4                                                                  |
| **Nextcloud Talk version**      | 20.1.0                                                                  |
| **`notifications` app enabled** | ✅ yes                                                                   |
| **`notify_push` app enabled**   | ✅ yes                                                                   |

#### Application config

```json
{
  "theme": "dark",
  "systemTitleBar": true,
  "monochromeTrayIcon": false,
  "zoomFactor": 1.1999999999999997,
  "playSoundChat": "respect-dnd",
  "playSoundCall": "respect-dnd",
  "enableCallbox": "respect-dnd"
}
```

It can be found in the About window (open from the menu or via <kbd>F1</kbd> and click on "Copy report" button).

</details>

<details>
<summary>Client logs</summary>

Press <kbd>Ctrl+Shift+I</kbd> (<kbd>Cmd+Option+C</kbd> on macOS) or use the context menu to open the developer tools, then go to the Console tab.

```
Not connected to signaling server yet, defer joining room m4bnafes

mydomainfonextcloud.net/ocs/v2.php/apps/user_status/api/v1/statuses/_Filip:1 
        
        
       Failed to load resource: the server responded with a status of 404 ()
```

</details>

**Comment by filipdot** [comment]
when i send it works, when somebody then not. Sometimes it works some times it doesn't.
My message:
![Image](https://github.com/user-attachments/assets/cfe4c3e1-233b-45ae-befb-7b245e9350da)
Somebody else:
![Image](https://github.com/user-attachments/assets/42876b96-a134-437b-82fe-e1f8c84b6a6b)

**Comment by Antreesy** [comment]
When you send it, we ue blob with local picture to not load of again (until restarted)

**Comment by filipdot** [comment]
i get 404 in browser when trying to generate preview
![Image](https://github.com/user-attachments/assets/0e60560f-8c7a-401c-a870-abfe4982df5e)

<html><body>
<span class="message-body-wrapper">
GET | https://mydomain/core/preview?fileId=177795&x=-1&y=32&a=1
-- | --


</span>
</body>
</html>

HTTP/1.1 404 
server: openresty
date: Thu, 19 Dec 2024 08:12:31 GMT
content-type: application/json; charset=utf-8
content-length: 2
cache-control: no-cache, no-store, must-revalidate
content-security-policy: default-src 'none';base-uri 'none';manifest-src 'self';frame-ancestors 'none'
feature-policy: autoplay 'none';camera 'none';fullscreen 'none';geolocation 'none';microphone 'none';payment 'none'
referrer-policy: no-referrer
strict-transport-security: max-age=31536000;
x-content-type-options: nosniff
x-frame-options: SAMEORIGIN
x-permitted-cross-domain-policies: none
x-request-id: Jm9wbPFpP1T81Ps47yhp
x-robots-tag: noindex, nofollow
x-xss-protection: 1; mode=block
X-Firefox-Spdy: h2

GET /core/preview?fileId=177795&x=-1&y=32&a=1 HTTP/1.1
Host: mydomain
User-Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:133.0) Gecko/20100101 Firefox/133.0
Accept: image/avif,image/webp,image/png,image/svg+xml,image/*;q=0.8,*/*;q=0.5
Accept-Language: pl,en-US;q=0.7,en;q=0.3
Accept-Encoding: gzip, deflate, br, zstd
DNT: 1
Sec-Fetch-Dest: image
Sec-Fetch-Mode: no-cors
Sec-Fetch-Site: same-origin
Connection: keep-alive
Cookie: __Host-nc_sameSiteCookielax=true; __Host-nc_sameSiteCookiestrict=true; cookieyes-consent=consentid:eU1LZEJrRzRuOWVCSWJ5ZThLaHIzbEdlU2dZaE8yMTE,consent:yes,action:no,necessary:yes,functional:yes,analytics:yes,performance:yes,advertisement:yes,other:yes; tk_or=%22https%3A%2F%2Finsights.hotjar.com%2F%22; tk_lr=%22%22; _gcl_au=1.1.1645389743.1730190742; _ga=GA1.1.437307742.1730190741; _ga_JLJYRJCMV7=GS1.1.1734534231.43.0.1734534231.60.0.0; _hjSessionUser_568947=eyJpZCI6IjBkOTgwZDM4LWJlNmEtNTYzNy1hYzlmLTNiM2I4MzMyMTVkMiIsImNyZWF0ZWQiOjE3MzA3OTM4MjQzODYsImV4aXN0aW5nIjp0cnVlfQ==; tk_ai=S3LaHUV%2BybnrVlaWiGlIK6lK; nc_username=Filip; nc_token=FXHTces%2B3EzcBp1G7X1FZgFGymWsP88e; nc_session_id=4b39f51bf3a27e5bab62ddcd09fac420; oc_sessionPassphrase=tbEL0%2BVhXbHOob6PBG%2FvK3T2S%2Ba19lIv7zWivBSIpu2M81a1328q5hqPxlJ%2F6wZRuT5G4PCvlnQJ2%2FCkhVYjkdjxwT2mQbBS8nMsIrVV%2FHvGoc3Q60ehYdACZ6GZ7lc9; oc8hbd6qyede=4b39f51bf3a27e5bab62ddcd09fac420

**Comment by alexhjelmnf** [comment]
Correct, we get this also. Works in mobile talk but not on desktop talk.

**Comment by frmoronari** [comment]
Hi;
I have same error, in Windows and Linux Desktop APP, in Android APP and in the Web Browser client direct on Nextcloud interface. 
Im searching for solution in web and / or awaiting for Nextcloud team for one solution.

**Comment by frmoronari** [comment]
Hi again;

For problem in Talk in browser, i have one solution.
Edit nextcloud config file "config.php", generally in "/web-server-root-directory/nextcloud/config/config.php" and adding these lines on end file:
```
array (
    'OC\Preview\PNG',
    'OC\Preview\JPEG',
    'OC\Preview\GIF',
    'OC\Preview\BMP',
    'OC\Preview\XBitmap',
    'OC\Preview\MP3',
    'OC\Preview\TXT',
    'OC\Preview\MarkDown',
    'OC\Preview\OpenDocument',
    'OC\Preview\Krita',
    'OC\Preview\HEIC',
  ),
```
before last ");" in file.

**Comment by frmoronari** [comment]
Hi again again;

One update, the solution above resolve all platforms.

This print is the Talk in Linux:

![Image](https://github.com/user-attachments/assets/3341bda6-f419-4b3a-b360-47d35b4e0f68)

Red is before config, Green is after config.

After edit file, i´m reload web server (apache2) in my server.

**Comment by tim-geier** [comment]
We also had this problem, no pictures in the desktop app but it worked fine in the mobile app and in browser.

Turns out the only problem was the setting 'enable_previews' in config.php. Changed this from false to  `'enable_previews' => true,` and the pictures started working right away.
I have no idea why this was turned off, it's an older instance that got updated over the years. Maybe there was a time when the default setting was false or someone turned off previews for performance reasons.

Maybe there should be a configuration warning if previews are turned off and Talk is enabled.
